include/seqan3/alignment/pairwise/detail/policy_affine_gap_recursion.hpp

```diff
@@ -147,11 +192,19 @@ class policy_affine_gap_recursion
      * initialised to 0, while the value of the horizontal and vertical matrix are initialised as:
      * \f$V[0, 0] = H[0, 0] = g_o\f$.
      */
-    affine_cell_type initialise_origin_cell() const noexcept
+    // affine_cell_type initialise_origin_cell() const noexcept
```

Comment:
needs to be reenabled after it is clear how we can disambiguate this function from the one below. Probably using a concept requirement on the affine_cell_type which should be defined conditionally using the configuration traits.